### PR TITLE
Topic2

### PR DESCRIPTION
**Pull request by facusuarez**
Finished Topic 2 (Without Modules)

### CODE REVIEW

File: facundo/02-oop-inheritance/index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<head >
```
Comment (marcelososa) [inline facundo/02-oop-inheritance/index.html]: Wrong HTML format, missing tags.

File: facundo/02-oop-inheritance/script.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
+window.addEventListener("load", function () {
+    //tryMovie();
```
Comment (marcelososa) [inline facundo/02-oop-inheritance/script.js]: Remove all commented code.

File: facundo/02-oop-inheritance/script.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    tryActors();
+})
+/**
+ * 
+ * 
+ * @class EventEmitter
+ * A class that implements an Observer Pattern, it Subscribes, emits, and unsubscribes
+ * from events.
+ */
+class EventEmitter {
```
Comment (marcelososa) [inline facundo/02-oop-inheritance/script.js]: You need to split this file in different modules, at least one file per class and create a final bundle with Babel.

File: README.md [hunk 4]
```diff
@@ -1,9 +0,0 @@
-# HTML5 BootCamp base folders layout
```
Comment (marcelososa) [inline README.md]: Why remove this file?

File: facundo/02-oop-inheritance/Movie.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+/**
+ * 
+ * 
+ * @class Movie
+ * @extends {EventEmitter}
+ * A Class that extends event emitter, it has the Clasic Movie attributes, and emits an event
+ * when play, resume and pause
+ */
+class Movie extends EventEmitter {
```
Comment (marcelososa) [inline facundo/02-oop-inheritance/Movie.js]: `Movie` class extends from `EventEmitter`, should include here that dependency.

File: facundo/02-oop-inheritance/index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
+import EventEmitter from './EventEmitter';
```
Comment (marcelososa) [inline facundo/02-oop-inheritance/index.js]: Is not necessary import this class, this dependency should be resolved in the `Movie` class.

Comment (facusuarez) [inline facundo/02-oop-inheritance/index.js]: Fixed, i just don't realize that.  Thank you for your review!

File: facundo/02-oop-inheritance/Movie.js (hunk 5 above)
Comment (facusuarez) [inline facundo/02-oop-inheritance/Movie.js]: Fixed

File: README.md (hunk 4 above)
Comment (facusuarez) [inline README.md]: Ops, Sorry, now it's back again.

File: facundo/02-oop-inheritance/script.js (hunk 3 above)
Comment (facusuarez) [inline facundo/02-oop-inheritance/script.js]: Fixed

File: facundo/02-oop-inheritance/script.js (hunk 2 above)
Comment (facusuarez) [inline facundo/02-oop-inheritance/script.js]: Fixed

File: facundo/02-oop-inheritance/index.html (hunk 1 above)
Comment (facusuarez) [inline facundo/02-oop-inheritance/index.html]: Fixed

File: README.md (hunk 4 above)
Comment (marcelososa) [inline README.md]: You need undo this deleted file, how?
```sh
# get the all changes from origin repo
git fetch origin
# move to the right branch
git checkout Topic2
# deleting the wrong file
git rm README.md.txt
# restoring the right file from the origin repository
git checkout origin/master -- README.md
# check modified files
git status
# add changes to commit
git add .
# create the new commit
git commit -m "Deleted README.md.txt and restore README.md"
# push the last chages to update the PR
git push personal Topic2
```

Comment (facusuarez) [inline README.md]: That works fine! Thanks Marce.

**Pull request merged**